app/components/primer/primer.pcss

```diff
@@ -3,3 +3,4 @@
 @import "./alpha/banner.pcss";
 @import './alpha/segmented_control.pcss';
 @import "./beta/button.pcss";
+@import "./beta/blankslate.pcss";
```

Comment:
If we import it here (to be used in Lookbook), I guess we'll have to expand the import on dotcom. Like https://github.com/github/github/pull/245966.

Or an alternative would be to not import `blankslate` here, but update the Lookbook import to something like:

```diff
- *= require @primer/css/dist/blankslate.css
+ *= require ../../../../../app/components/primer/beta/blankslate.css
```

But I haven't figured out the exact local path.

Reply:
nvm.. We decided to not expand the PVC imports on dotcom. It might cause regression changing the source order. But we'll have to do that eventually anyways.